libs/sdk-ui-loaders/src/dashboard/dashboardLoader.tsx

```diff
@@ -217,7 +218,9 @@ export class DashboardLoader implements IDashboardLoader {
 
         const additionalPlugins = await initializeEmbeddedPlugins(ctx, this.embeddedPlugins);
         const loadedPlugins = await initializeLoadedPlugins(ctx, plugins);
-        const allPlugins = [...loadedPlugins, ...additionalPlugins];
+        // Omit plugins that do not match the loaded dashboard engine version.
+        const compliantLoadedPlugins = omitNonCompliantPlugins(engine, loadedPlugins);
```

Comment:
Do we want to do this? I wonder if we should hard stop in this case and not load anything (similar to when a plugin is from not allowed origin - then all the plugin loading is stopped).

Reply:
Updated